src/components/Charts/ChartCaseDensity.tsx

```diff
@@ -62,14 +62,22 @@ const ChartCaseDensity: FunctionComponent<{
 
   const data: Point[] = columnData.filter(hasData);
 
+  const lastPoint = last(data);
+  const activeZone = getZoneByValue(getY(lastPoint), zones);
+
   const dates = data.map(getDate);
   const minDate = d3min(dates) || new Date('2020-01-01');
   const currDate = new Date();
   const xScale = getZonesTimeScale(minDate, currDate, 0, chartWidth);
 
   const yDataMax = d3max(data, getY) || 100;
   const yAxisLimits = getAxisLimits(0, yDataMax, zones);
-  const [yAxisMin, yAxisMax] = [-9, Math.min(yAxisLimits[1], capY)];
+
+  // Adjusts the min y-axis to make the Low label fit only if
+  // the current level is Low
+  const isLow = activeZone.level === Level.LOW;
+  const yAxisMin = isLow ? -9 : 0;
```

Comment:
Do you think it's worth tweaking this `-9` down a bit so that "Low" still fits but there's a bit less extra padding?

Not a big deal at this point, so feel free to punt.

Reply:
`-6` seems to be better, updating!